src/Libraries/RevitNodes/Elements/FloorType.cs

```diff
@@ -8,77 +11,51 @@ namespace Revit.Elements
     /// A Revit FloorType
     /// </summary>
     /// http://revitapisearch.com.s3-website-us-east-1.amazonaws.com/html/b6fd8c08-7eea-1ab4-b7ab-096778b46e8f.htm
```

Comment:
is this from us or an internal Autodesk thing ? I get a `403` not authorised error

Reply:
Autodesk thing